chutney/server/src/test/java/com/chutneytesting/execution/domain/purge/PurgeServiceTest.java

```diff
@@ -223,6 +253,49 @@ void purge_oldest_when_limit_exceeded() {
         }
     }
 
+    @Nested
+    @DisplayName("deletes empty campaign's execution")
+    class CleanEmptyCampaignExecution {
+        @Test
+        void empty_campaign_execution() {
+            // Given
+            // One campaign's execution without scenarios' executions
+            // And a configuration limit set to 0 for campaigns' executions
+            int maxCampaignExecutionsConfiguration = 0;
```

Comment:
0 should remove all execution. Not revelant to check if the empty campaign is removed

Reply:
Added just a comment to underline the fact that the campaign is not deleted without code correction due to the "keep last success no matter what" rule.